docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jupyter/datascience-notebook:latest
+FROM jupyter/datascience-notebook:fa77fe99579b
```

Comment:
As described on the PR description, same reason as we have versioned datasets: reproducibility 😄 as it just happened, we don't want a potential future contributor to have trouble with his / her environment because some change that was introduced in a newer release.